client/src/classes/PeerProcessor.ts

```diff
@@ -168,7 +189,7 @@ export class PeerProcessor {
   getVolume(selfPosition: [number, number], peerPosition: [number, number]): number {
     const distance = Math.sqrt(
       Math.pow(selfPosition[0] - peerPosition[0], 2) +
-      Math.pow(selfPosition[1] - peerPosition[1], 2)
+        Math.pow(selfPosition[1] - peerPosition[1], 2)
```

Comment:
For later refactoring, this can be part of the `utils.ts`

Reply:
I agree

Reply:
- #251